synapse/lib/storm.py

```diff
@@ -245,6 +245,8 @@ def getArgParser(self):
     async def execStormCmd(self, runt, genr):
         ''' Abstract base method '''
         raise s_exc.NoSuchImpl('Subclass must implement execStormCmd')
+        for item in genr:
```

Comment:
????

Reply:
This method is not identified as an async generator when not implemented, so rather than getting a proper error message, you get a "coroutine has no __aiter__ method" type thing.  This just ensures it can actually raise the exception.